KiBuzzard/dialog/dialog.py

```diff
@@ -58,7 +62,12 @@ def __init__(self, parent, config, buzzard, func):
         self.m_LayerComboBox.SetSelection(0)
         self.m_HeightUnits.SetLabel("mm")
         self.m_WidthUnits.SetLabel("mm")
+<<<<<<< Updated upstream
```

Comment:
Merge leftovers

Reply:
Ah, I think this is GitHub Desktop's fault. I'll go fix things...